cmd/instance_pool_show.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type instancePoolItemOutput struct {
+	ID              string                     `json:"id"`
+	Name            string                     `json:"name"`
+	Description     string                     `json:"description"`
+	Serviceoffering string                     `json:"serviceoffering"`
+	Template        string                     `json:"templateid"`
+	Zone            string                     `json:"zoneid"`
+	Affinitygroups  []string                   `json:"affinitygroups"`
+	Securitygroups  []string                   `json:"securitygroups"`
+	Privnets        []string                   `json:"Privnets"`
+	Keypair         string                     `json:"keypair"`
+	Size            int                        `json:"size"`
+	State           egoscale.InstancePoolState `json:"state"`
+	Virtualmachines []string                   `json:"virtualmachines"`
+}
+
+func (o *instancePoolItemOutput) toJSON()  { outputJSON(o) }
+func (o *instancePoolItemOutput) toText()  { outputText(o) }
+func (o *instancePoolItemOutput) toTable() { outputTable(o) }
+
+var instancePoolShowCmd = &cobra.Command{
+	Use:     "show <name | id>",
+	Short:   "Show an instance pool",
```

Comment:
The command is missing the `Long` field with custom output annotations, see how other `show` commands are implemented for reference.